FBSnapshotTestCase/FBSnapshotTestController.h

```diff
@@ -41,6 +51,11 @@ extern NSString *const FBReferenceImageFilePathKey;
 @property (readwrite, nonatomic, assign) BOOL recordMode;
 
 /**
+ Save reference/captured/diffed images to disk after failures. Defaults to YES.
+ */
+@property (readwrite, nonatomic, assign) BOOL saveFailuresToDisk;
```

Comment:
hmm let's avoid this property for now, generally I don't mind the functionality added here but it's a bool property that would depend on some `if testRunsOnDevice`. This I would like to avoid.